docs/getting-started.stories.mdx

```diff
@@ -31,7 +31,7 @@ npm install carbon-react
 You will need to install the following dependencies in your project, these are peer-dependencies of carbon-react and are required.
 
 ```sh
-npm install react@^16.12.0 react-dom@^16.12.0 styled-components@^4.4.1
+npm install @sage/design-tokens@^2.3.0 react@^16.12.0 react-dom@^16.12.0 styled-components@^4.4.1
```

Comment:
Sorry @Parsium not your changes I know - can you please update the React deps here to be `16.14.0` while you're changing this line anyway - we require those as peer deps now

```suggestion
npm install @sage/design-tokens@^2.3.0 react@^16.14.0 react-dom@^16.14.0 styled-components@^4.4.1
```